graphql-cli/src/test/java/io/ballerina/graphql/generator/ballerina/ServiceTypesGeneratorTest.java

```diff
@@ -699,15 +766,71 @@ public void testGenerateSrcForSchemaWithFileUploadFields() {
             ServiceTypesGenerator serviceTypesGenerator = new ServiceTypesGenerator();
             serviceTypesGenerator.setFileName(fileName);
             String generatedServiceTypesContent = serviceTypesGenerator.generateSrc(graphQLSchema);
-            writeContentTo(generatedServiceTypesContent, this.tmpDir);
+            writeContentTo(generatedServiceTypesContent, this.tmpDir, TYPES_FILE_NAME);
+
+            Path expectedServiceTypesFile =
+                    resourceDir.resolve(Paths.get("serviceGen", "expectedServices", expectedFile));
+            String expectedServiceTypesContent = readContentWithFormat(expectedServiceTypesFile);
+            String writtenServiceTypesContent = readContentWithFormat(this.tmpDir.resolve("types.bal"));
+            Assert.assertEquals(expectedServiceTypesContent, writtenServiceTypesContent);
+        } catch (ValidationException | IOException | ServiceGenerationException e) {
+            Assert.fail(e.getMessage());
+        }
+    }
+
+    @Test(
+            groups = {"service-type-for-objects"},
+            description = "Test for schema with input type fields having default values"
```

Comment:
I don't think these descriptions add much value. Function name is descriptive enough.